tutorials/tracer_transport.ipynb

```diff
@@ -6,7 +6,11 @@
    "source": [
     "# Assembly of system with multiple domains, variables and numerics\n",
     "\n",
-    "This tutorial demonstrate how a transient problem may be solved in PorePy. We consider the purely advective tracer transport problem for a static flow field, see the tutorial darcy_and_transport_equations.ipynb. The geometry and flow parameters are as in example 2 of this [benchmark study](https://www.sciencedirect.com/science/article/pii/S0309170817300143).\n"
+    "This tutorial demonstrates how a transient problem may be solved in PorePy. \n",
+    "We consider the advective-diffusive tracer transport problem for a slightly compressible flow field described in [this tutorial](darcy_and_transport_equations.ipynb).\n",
```

Comment:
I get an error message when following the link on GitHub, presumably something wrong with the way notebooks are rendered on GH, but I think this is okay.